src/components/InputTypes/MultiSelect.jsx

```diff
@@ -18,7 +18,7 @@ const MultiSelect = (props) => {
   function renderOptions() {
     return Object.keys(options).map((option) => {
       return (
-        <MenuItem key={option} value={option}>
+        <MenuItem key={option} value={options[option]}>
```

Comment:
I learned that to make react more concise about keys in map functions, you should do something like add the additional index parameter to the map ex Object.keys(options).map((option, i) and then make the key={`multi_select_${option}_${i}`}.
Although it's unlikely that there would be a key that would be equal to "Female" you want to make the key as close to what it is rendering alongside the additional i parameter in map

Reply:
I tried this out, but I don't think it's necessary.

My reason is that when I tried to add a key in the `questionList.json` file with the same name as another, lint picked it up. Lint will catch any duplicate names we have in the keys within a single object, so uniqueness is practically guaranteed. Unless we want to also check for unique keys between different objects, which it won't pick up.

Reply:
Lint returns `Do not use Array index in keys` when attempting to add the index number as part of the key.

![Screen Shot 2019-10-03 at 7 52 22 PM](https://user-images.githubusercontent.com/35314961/66177707-684ca300-e617-11e9-91a4-50f6a6a4a857.png)

So I will not be doing the map index change.